packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -90,9 +90,13 @@ const ProjectForm: FC<Props> = ({
             <SettingsGridCard>
                 <div>
                     {warehouse && getWarehouseIcon(warehouse)}
-                    <LeftPanelTitle>
-                        <H5>Warehouse connection</H5>
-                        <DocumentationHelpButton href="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#warehouse-connection" />
+                    <LeftPanelTitle
+                        style={{ display: 'flex', alignItems: 'center' }}
+                    >
+                        <Title order={5} style={{ marginRight: '0px' }}>
+                            Warehouse connection
+                        </Title>
+                        <DocumentationHelpButton />
```

Comment:
This is great, but we could replace `LeftPanelTitle` with `Mantine`'s one, so: 

```tsx
 <Flex align="center">
                        <Title order={5} style={{ marginRight: '0px' }}>
                            Warehouse connection
                        </Title>
                        <DocumentationHelpButton />
                    </Flex>
```

Reply:
Thanks for the review @joaoviana , I will do it.